docs/pages/docs/features/caching.mdx

```diff
@@ -129,7 +129,9 @@ Luckily, you can control `turbo`'s cache fingerprinting (a.k.a. hashing) behavio
 }
 ```
 
-Pro Tip: In most monorepos, you're don't often use environment variables in shared packages—mostly only in applications. Thus, to get higher cache hit rates, you should likely only include environment variables in the package tasks where they are used.
+<Callout type="info">
+  Pro Tip: In most monorepos, you're don't often use environment variables in shared packages—mostly only in applications. Thus, to get higher cache hit rates, you should likely only include environment variables in the package tasks where they are used.
```

Comment:
Hi Jared !
Thank you for this PR and for turborepo 😃
This « pro-tip » seems to apply to each of our monorepo : perfect !

But something is unclear for me : 
- By « package task » do you mean the task with a name containing a « # » like « build#app » which overrides « build » in the pipeline ? 
I’m sorry but I couldn’t find a section about them in the docs.
- If true, in the example you mention an override but you specify all the props again.
Would it work if the override specified only dependsOn and the output was inherited from tle main build task ?

Reply:
Correct. (e.g. `web#build`). At the moment, there is not inheritance, so you must redeclare all keys in the pipeline

Reply:
Thanks Jared 👍🏻 
...and, btw, I really dig your rework with the environment variables specified directly in the `dependsOn` key instead of the new `env` array 🔥

Reply:
Thanks! As you know, we're trying to keep our API footprint as small as possible!